app-games/build.gradle.kts

```diff
@@ -0,0 +1,264 @@
+  id(GradlePluginId.HILT)
+}
+
+android {
+  namespace = "cm.aptoide.pt.app_games"
+
+  defaultConfig {
+    applicationId = "cm.aptoide.pt.app_games"
+    versionCode = AndroidConfig.VERSION_CODE
+    versionName = AndroidConfig.VERSION_NAME
+
+    buildConfigField("String", "MARKET_NAME", "\"apps\"")
+    buildConfigField("String", "STORE_DOMAIN", "\"https://ws75.aptoide.com/api/7.20221201/\"")
+    buildConfigField("String", "SEARCH_BUZZ_DOMAIN", "\"https://buzz.aptoide.com:10002\"")
+    buildConfigField(
+      type = "String",
+      name = "APTOIDE_WEB_SERVICES_APICHAIN_BDS_HOST",
+      value = "\"https://apichain.blockchainds.com/\""
+    )
+
+    testInstrumentationRunner = AndroidConfig.TEST_INSTRUMENTATION_RUNNER
+
+    manifestPlaceholders["dataPlaceholder"] = generateData()
+
+    buildConfigFieldFromGradleProperty("ROOM_SCHEMA_VERSION")
+    buildConfigFieldFromGradleProperty("ROOM_DATABASE_NAME")
+  }
+
+  signingConfigs {
+    create("signingConfigRelease") {
+      storeFile = file(project.properties[KeyHelper.KEY_STORE_FILE].toString())
+      storePassword = project.properties[KeyHelper.KEY_STORE_PASS].toString()
+      keyAlias = project.properties[KeyHelper.KEY_ALIAS].toString()
+      keyPassword = project.properties[KeyHelper.KEY_PASS].toString()
+    }
+  }
+
+  flavorDimensions.add(0, "mode")
+
+  productFlavors {
+    create("dev") {
+      dimension = "mode"
+      applicationIdSuffix = ".dev"
+      versionName = AndroidConfig.VERSION_NAME + "." + getDate()
+      versionCode = AndroidConfig.VERSION_CODE
+    }
+
+    create("prod") {
+      signingConfig = signingConfigs.getByName("signingConfigRelease")
+      dimension = "mode"
+    }
+  }
+
+  applicationVariants.all {
+    val variant = this
+    variant.mergedFlavor.manifestPlaceholders["dataPlaceholder"] = generateData()
+
+    variant.outputs.forEach { output ->
+      output.processManifestProvider.get().doLast {
+        val placeholders = variant.mergedFlavor.manifestPlaceholders
+        if (placeholders.isNotEmpty()) {
+          val buildType = "${variant.flavorName}${variant.buildType.name.capitalize()}"
+          val manifestDir = "app/build/intermediates/merged_manifests/$buildType"
+          val manifestFilePath = "$manifestDir/AndroidManifest.xml"
+          val manifestFile = File(manifestFilePath)
+
+          var manifestContent = manifestFile.readText()
+          placeholders.forEach { (key, value) ->
+            val pattern = Pattern.compile(Pattern.quote("<!-- \${$key} -->"), Pattern.DOTALL)
+            manifestContent = pattern.matcher(manifestContent).replaceAll(value as String?)
+          }
+          manifestFile.writeText(manifestContent)
+        }
+      }
+
+    }
+
+    variant.outputs
+      .map { it as com.android.build.gradle.internal.api.BaseVariantOutputImpl }
+      .forEach { output ->
+        val outputFileName =
+          "vanilla_${variant.baseName}_${variant.versionName}_${variant.versionCode}.apk"
```

Comment:
Add the apk signing status to the output file name. You can check the GH gradle to see how to do this